Userland/DevTools/HackStudio/Editor.cpp

```diff
@@ -261,7 +261,7 @@ void Editor::mousemove_event(GUI::MouseEvent& event)
         set_override_cursor(m_hovering_clickable && event.ctrl() ? Gfx::StandardCursor::Hand : Gfx::StandardCursor::IBeam);
 
     for (auto& span : document().spans()) {
-        bool is_clickable = (highlighter->is_navigatable(span.data) || highlighter->is_identifier(span.data));
+        bool is_clickable = (highlighter->is_navigable(span.data) || highlighter->is_identifier(span.data));
```

Comment:
According to wiktionary, "navigatable" is legitimate. I also think that it's easier to understand than "navigable" if you've never come accross the word before.